src/main/webapp/index.html

```diff
@@ -10,41 +10,133 @@
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>          <!-- Ajax, Jquery -->
     <script defer src="https://use.fontawesome.com/releases/v5.3.1/js/all.js"></script>             <!-- Font awesome- used for icons-->
     <script src="script.js"></script>
+    <script src="keys.js"></script>
+
     <link rel="stylesheet" href="style.css">
+
   </head>
 
-  <body onload="javascript:getUrl()">
+  <body>
 
-    <!-- This is the first section of the page that contains the navbar containing the WeTime title and other home, log in buttons, etc.. -->
-    <section class="hero is-default is-bold">
-      <div class="hero-head">
-        <nav class="navbar">
-          <div class="container">
+    <!-- This is the first section of the page that contains the navbar containing the WeTime title and other home, log in, log out buttons, etc.. -->
+    <!-- Navbar. --> 
+    <section class="section">
+      <div class = "container">
+        <nav class="navbar is-spaced is-dark" role="navigation" aria-label="main navigation">
+          
+          <div class="navbar-brand">
+            <a class="navbar-item" href="index.html"> WeTime <i class="fas fa-heartbeat"></i> </a>
 
-            <div class="navbar-brand">
-              <a class="navbar-item" href="index.html"> WeTime <i class="fas fa-heartbeat"></i> </a>
-              <span class="navbar-burger burger" data-target="navbarMenu"><span></span><span></span><span></span></span>
+            <a role="button" class="navbar-burger burger" aria-label="menu" aria-expanded="false" data-target="navbarBasicExample">
+              <span aria-hidden="true"></span>
+              <span aria-hidden="true"></span>
+              <span aria-hidden="true"></span>
+            </a>
+          </div>
+
+          <!-- Navbar options to other other html pages. --> 
+          <div id="navbarBasicExample" class="navbar-menu">
+            <div class="navbar-start">
+              <a class="navbar-item"><span class="icon"><i class="fas fa-home"></i></span><span> Home</a>
```

Comment:
Do we need a home button here or can the logo be the home button? so the nav bar is less crowded... :)

Reply:
That's good. I will change that now.